.githooks/pre-push

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+remote="$1"
+url="$2"
+
+z40=0000000000000000000000000000000000000000
```

Comment:
What is this for?

Reply:
It was taken from the sample in my .git/hooks/pre-push.sample

Reply:
Apparently it is used to detect branch deletion